fluxc/src/main/java/org/wordpress/android/fluxc/store/PostStore.java

```diff
@@ -695,6 +696,19 @@ private void handleFetchPostsCompleted(FetchPostsResponsePayload payload) {
         emitChange(onPostChanged);
     }
 
+    private void handleFetchSinglePostCheckLocallyChanged(FetchPostResponsePayload payload) {
+        // Process a fetch action only for a post if it hasn't local changes
+        // as we'd otherwise overwrite and lose these local changes forever
+        PostModel localPost = getPostByRemotePostId(payload.post.getRemotePostId(), payload.site);
+        if (localPost != null) {
```

Comment:
How do you feel about combining these `if`'s and the `else` statement into a single one? I think with the comment above it, it would be fine to do so.

```suggestion
        if (localPost == null || !localPost.isLocallyChanged()) {
            updatePost(payload.post, false);
        }
```

We could add `if it doesn't exist or if doesn't have local changes` to the above comment to make it even more clearer.